torch/csrc/distributed/c10d/FlightRecorder.cpp

```diff
@@ -1,21 +1,23 @@
-// TODO: Make Fligth Recorder device agnostic
 #ifdef USE_C10D_NCCL
```

Comment:
Why do we still have `#ifdef USE_C10D_NCCL` here?

Reply:
because #include <cuda_runtime.h>